src/lib/components/Swap/Status/StatusDialog.tsx

```diff
@@ -12,7 +12,7 @@ import { ExplorerDataType } from 'utils/getExplorerLink'
 import ActionButton from '../../ActionButton'
 import Column from '../../Column'
 import Row from '../../Row'
-import Summary from '../Summary'
+import SwapSummary from '../Summary'
```

Comment:
renamed to help make clear the behavior below where this is only used for `SWAP` transaction types. (not wrap or approvals)

Reply:
Fan of this - it always confused me what this was

Reply:
Should you also rename the file?

Reply:
I chose not to because it's already in the `components/Swap/Summary` directory. Making it `components/Swap/SwapSummary` feels weirdly repetitive and none of the other components follow that pattern. I do think changing the import could help though! I'll do that.